js/main.js

```diff
@@ -0,0 +1,219 @@
+  createElement('headCon', 'div', 'container-fluid text-white', 'headCon', null, body, '5CAST', 'HC');
+  createElement('subHead', 'div', 'container-fluid text-white', 'subHead', null, body, '"Better than a forecast"', 'subHead');
+  createElement('zipCon', 'div', 'container d-flex', 'zipCon', null, body, null, 'ZC');
+    createElement('input', 'input', 'col-3 zipLine', 'input', 'number', ZC, null, 'input');
+    createElement('submit', 'button', 'col-3 zipLine', 'goButton', null, ZC, 'Get your 5CAST!', 'goButton');
+  createElement('loCon', 'div', 'container d-flex', 'locCon', null, body, null, 'LC');
+    createElement('locRow', 'div', 'row', 'locRow', null, LC, null, 'LR');
+      createElement('locCol', 'div', 'col', 'lCol', null, LR, null, 'LCL')
+        createElement('tCard', 'div', 'card', 'lCard', null, LCL, null, 'lCard');
+          createElement('cCardTitle', 'p', 'card-header', 'cCardTitle', null, lCard, 'LOCATION:', 'lCardTitle')
+          createElement('loc', 'p', 'card-text li', 'loc', null, lCard, null, 'loc');
+  createElement('tempCon', 'div', 'container d-flex', 'tempCon', null, body, null, 'TC');
+    createElement('tempRow', 'div', 'row', 'tempRow', null, TC, null, 'TR');
+      createElement('tempCol', 'div', 'col', 'tCol', null, TR, null, 'TCL')
+        createElement('tCard', 'div', 'card', 'tCard', null, TCL, null, 'tCard');
+          createElement('tCardTitle', 'p', 'card-header', 'tCardTitle', null, tCard, 'TEMPERATURE:', 'tCardTitle')
+            createElement('tempList', 'ul', 'list-group list-group-flush li', 'tempList', null, tCard, null, 'tempList' )  
+              createElement('tempK', 'li', 'list-group-item li', 'tempK', null, tempList, null, 'tempK');
+              createElement('tempF', 'li', 'list-group-item li', 'tempF', null, tempList, null, 'tempF');
+              createElement('tempC', 'li', 'list-group-item li', 'tempC', null, tempList, null, 'tempC');
+  createElement('conCon', 'div', 'container d-flex', 'conCon', null, body, null, 'CC');
+    createElement('conRow', 'div', 'row', 'conRow', null, CC, null, 'CR');
+      createElement('conCol1', 'div', 'col', 'cCol', null, CR, null, 'CCL')
+        createElement('cCard', 'div', 'card', 'cCard', null, CCL, null, 'cCard');
+          createElement('cCardTitle', 'p', 'card-header', 'cCardTitle', null, cCard, 'CONDITION:', 'cCardTitle')
+            createElement('condition', 'p', 'card-text li', 'condition', null, cCard, null, 'condition');
+  createElement('popUp', 'div', 'popUp card', 'popUp', null, body, null, 'popUp');
+    createElement('popUpText', 'h1', 'popUpText card-header', 'PUT', null, popUp, 'INVALID ZIPCODE', 'PUT');
+    createElement('popUpButton', 'button', 'btn btn-danger', 'PUB', null, popUp, 'Come on, you can do it.', 'PUB');
+
+  goButton.addEventListener('click', function() {
+    let zipcode = input.value
+    let API = `https://api.openweathermap.org/data/2.5/weather?zip=${zipcode}&appid=${APIKey}`;
```

Comment:
Would be easier to build the API call in the getWeatherData function. I think it makes more sense there. That would move the zip as the only variable to that function.